### PR TITLE
Feature: User should have an interface to enter a time or period in minutes

### PR DESCRIPTION
**Pull request by saszaCodes**
#### What does this PR do?
It adds the interface for the user to enter time in minutes and methods to user's input before submitting. It uses React Context so it also server as a proof of concept of how I would use it in the entire app

#### Description of Task to be completed?
Add an interface for the user to input time in minutes

#### How should this be manually tested?
Input various numbers in the input field and try to submit them

#### Any background context you want to provide?
All the warnings detected by Hound CI are due to a bad setup of ESLint, which is fixed by another PR. If any conflicts arise with this branch after merging the `Fix: Resolve ESLint warnings` branch with main, I will fix them before merging `Feature: Interface to enter time` into main

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,42 +1,50 @@
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import { useState } from 'react';
```
Comment (Avuidrauxs) [inline src/App.js:1]: I agree

File: src/Contexts/TimerContext.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const TimerContext = createContext();
+
+export default TimerContext;
```
Comment (Avuidrauxs) [inline src/Contexts/TimerContext.js:5]: ### Nitpick

Add a new line.

**Comment by Avuidrauxs** [comment]
So, I just checked out the UX of this feature. Great job, but I think we need more improvements. Here are my recommendations.

- ##### Input Validation

Screen Shot 2021-07-25 at 7 47 33 AM

I believe we should rather not allow users to be able to type text in there using some sort of control, maybe try regular expressions.


- ##### Validation prompt

Screen Shot 2021-07-25 at 7 47 45 AM

This should dismiss when the input is cleared and also add a placeholder text to inform users what the limits are, rather than let them make mistakes before they know the right thing.

**Review by saszaCodes — COMMENTED** [review]
@Avuidrauxs ok, I fixed errors indicated by you and Hound

**Comment by saszaCodes** [comment]
@Avuidrauxs I implemented the rest of the changes you requested, in my opinion the branch is ready to be merged

**Review by Avuidrauxs — APPROVED** [review]

**Pull request merged**